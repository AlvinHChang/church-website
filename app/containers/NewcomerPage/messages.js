/*
 * NewcomerPage Messages
 *
 * This contains all the text for the NewcomerPage container.
 */
import { defineMessages } from 'react-intl';

export const scope = 'app.containers.NewcomerPage';

export default defineMessages({
  header: {
    id: `${scope}.header`,
    defaultMessage: 'This is the NewcomerPage container!',
  },
});
