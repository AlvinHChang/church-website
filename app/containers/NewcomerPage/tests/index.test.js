import React from 'react';
import { render } from 'react-testing-library';
import { IntlProvider } from 'react-intl';

import NewcomerPage from '../index';

describe('<NewcomerPage />', () => {
  it('should render and match the snapshot', () => {
    const {
      container: { firstChild },
    } = render(
      <IntlProvider locale="en">
        <NewcomerPage />
      </IntlProvider>,
    );
    expect(firstChild).toMatchSnapshot();
  });
});
