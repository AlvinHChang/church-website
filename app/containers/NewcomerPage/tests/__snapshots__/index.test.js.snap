// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<NewcomerPage /> should render and match the snapshot 1`] = `
<h1>
  <span>
    This is the NewcomerPage container!
  </span>
</h1>
`;
